changelog.txt

```diff
@@ -1,3 +1,29 @@
+1.9.0-beta3 (2019-02-24)
+------------------------------------------------------------------------
+- Feature: Option to adjust font size separately from GUI size (#7003)
+- Feature: Increase maximum number of orders from 64000 to ~16.7m (#7220)
+- Add: Show performance of AI and GS in framerate window
```

Comment:
I'm not seeing any mention of the framerate window in beta1, should this be here?

Reply:
- Feature: Framerate display window (#6822)